lib/algo/memoization/test.js

```diff
@@ -0,0 +1,40 @@
+const chai = require('chai');
+const knapsack = require('./knapsack');
+
+function Item(profit, volume) {
+  this.profit = profit;
+  this.volume = volume;
+
+  this.getProfit = function () {
+    return this.profit;
+  };
+  this.getVolume = function () {
+    return this.volume;
+  };
```

Comment:
`getProfit` & `getVolume` is not making any improvement as `profit` & `volume` is publicly available. Don't use a constructor if you don't need it.

BTW, is `profit` & `volume` a standard construct? Or they can be generalized.